.env.development.local.example

```diff
@@ -10,3 +10,4 @@ VITE_OIDC_AUDIENCES=kukkuu-api-dev,profile-api-test
 VITE_ADMIN_TICKET_VALIDATION_URL=https://kukkuu-admin-ui.test.hel.ninja/check-ticket-validity
 VITE_CMS_URI=https://kukkuu.app-staging.hkih.hion.dev/graphql
 VITE_HELSINKI_PROFILE_URL=https://profiili.test.hel.ninja/loginsso
+VITE_IDLE_TIMEOUT_IN_MS=3600000
```

Comment:
```suggestion
VITE_IDLE_TIMEOUT_IN_MS=3600000  # 60min = 60 * 60 * 1000ms
```

Reply:
And similarly some comments in other places would be good IMHO for clarity.

Reply:
comments in which places? I mean, for me it looks more clear as i ve done that, so if you point out what to comment, i will definitely do. 3600000 value we had already without a comment in other variable in use, so i thought its ok